package.json

```diff
@@ -57,8 +56,7 @@
     "typescript": "^5.1.6"
   },
   "peerDependencies": {
-    "@effect/data": ">=0.12 <1",
-    "@effect/io": ">=0.31 <1",
+    "effect": ">=2.0.0-next.23 <3",
```

Comment:
I can't tell with certainty because I don't know what the different package managers use to compare/parse package versions. However, if we assume they use `semver` or something with similar capabilities/quality, then I think it should work. See some experimentation I did with `semver` in a Node REPL:

<img width="380" alt="Screenshot 2023-08-02 at 16 12 07" src="https://github.com/iamchanii/pothos-plugin-effect/assets/5737996/38258827-1ad8-48e0-b89f-d130f5085062">